composer.json

```diff
@@ -13,19 +13,24 @@
         "bin-dir": "bin"
     },
     "require": {
-        "php": "^7.4",
-        "predis/predis": "^1.1.1",
+        "php": ">=8.0",
+        "predis/predis": "^1.1.8",
         "doctrine/cache": "~1.3",
-        "b1rdex/predis-compressible": "^0.5.0"
+        "b1rdex/predis-compressible": "^1.0",
+        "symfony/cache": "~5.0 || ~6.0",
+        "symfony/config": "~5.0 || ~6.0",
+        "symfony/dependency-injection": "~5.0 || ~6.0",
+        "symfony/http-foundation": "~5.0 || ~6.0",
+        "symfony/http-kernel": "~5.0 || ~6.0",
+        "symfony/yaml": "~5.0 || ~6.0"
     },
     "require-dev": {
-        "m6web/php-cs-fixer-config": "2.0.*",
-        "atoum/atoum": "^3.3.0",
-        "atoum/atoum-bundle": "^2.0",
+        "m6web/php-cs-fixer-config": "^2.0",
+        "atoum/atoum": "^4.0.3",
         "atoum/stubs": "~2.5",
         "m6web/redis-mock": "5.2.*",
-        "symfony/symfony": "~5.0",
-        "psr/cache": "~1.0"
+        "psr/cache": "~1.0",
+        "friendsofphp/php-cs-fixer": "^3.4"
```

Comment:
```suggestion
```

Reply:
Min version for php 8.1, see this failed ci : https://github.com/BedrockStreaming/RedisBundle/actions/runs/2232153155

Reply:
ok, we probably have to fix this directly in `m6web/php-cs-fixer-config` then

it is fine to merge as is and remove it later when this will be fixed 👌